h/static/scripts/app.coffee

```diff
@@ -4,9 +4,8 @@ if window.RAVEN_CONFIG
   require('./raven').init(window.RAVEN_CONFIG)
 
 
-require('autofill-event')
 angular = require('angular')
-require('angular-jwt')
+require('autofill-event')
```

Comment:
TODO - Add a note here that `autofill-event` needs to be included _after_ `window.angular` is defined by `require('angular')`